### ISSUE TITLE
WARNING: name mapping not found

### ISSUE THREAD

**Issue by SplinterGU** [goatfungus/NMSSaveEditor#790]
WARNING: name mapping not found: SQw
  WARNING: name mapping not found: Naj
  WARNING: name mapping not found: O;Y
  WARNING: name mapping not found: =Wk

**Comment by AlucardTheRed9** [comment]
Yeah, I have that too. Doesnt seem like anything bad is happening.... Hm.

**Comment by Devilin-Pixy** [comment]
These cause no issue, just lack mapping to an understandable name
```
SQw = WhaleEvent
Naj = OverriddenReward
O;Y = PageIdx
=Wk = LastSeenEntryIdx
```

**Comment by SplinterGU** [comment]
> These cause no issue, just lack mapping to an understandable name
> 
> ```
> SQw = WhaleEvent
> Naj = OverriddenReward
> O;Y = PageIdx
> =Wk = LastSeenEntryIdx
> ```
> 
> I am sure these will be included with a future update

Thanks... I add more info...


	"SeenStories":[
		{
			"PagesData":[
				{
					"O;Y":3,
					"=Wk":1
				},
				{
					"O;Y":0,
					"=Wk":8
				},
				{
					"O;Y":2,
					"=Wk":0
				}
			]
		},
		
		
		
		* FleetExpeditions/Events
		
						"Events":[
					{
						"Seed":[
							true,
							"0xC699C9DEE9CA7ED5"
						],
						"UA":0,
						"AffectedFrigateIndices":[],
						"RepairingFrigateIndices":[],
						"AffectedFrigateResponses":[],
						"EventID":"^MINING_3",
						"InterventionEventID":"^INT_DIPLOMAT_1",
						"OverriddenDescription":"",
						"SQw":false,
						"Naj":"^",
						"Success":true,
						"IsInterventionEvent":false,
						"AvoidedIntervention":false
					},
